pithos/pithos.py

```diff
@@ -1039,6 +1068,7 @@ def on_prefs_response(self, widget, response):
 
         email = self.settings.get_string('email')
         if response == Gtk.ResponseType.APPLY:
+            self.worker_run(self.on_explicit_content_filter_checkbox)
```

Comment:
Only talking with the Pandora API should be ran in a thread, these calls are modifying the UI from a thread, big no-no.